app/models/user.rb

```diff
@@ -108,7 +108,7 @@ def pending_profile
 
   def fraud_review_eligible?
     return false if !fraud_review_pending?
```

Comment:
`Profile#fraud_review_pending?` now simply returns `Profile#fraud_review_pending_at.present?` so the `fraud_review_pending_at&` in the next line now makes this line redundant.